appveyor.yml

```diff
@@ -1,18 +1,27 @@
 ---
 version: "{build}"
 clone_depth: 5
+init:
+  - set PATH=C:\ruby%RUBY_VERSION%\bin;C:\Program Files\7-Zip;C:\Program Files\AppVeyor\BuildAgent;C:\Program Files\Git\cmd;C:\Windows\system32;C:\Program Files;C:\Windows
+  - if %ruby_version%==_trunk (
+      appveyor DownloadFile https://dl.bintray.com/msp-greg/ruby_windows/ruby_trunk.7z -FileName C:\ruby_trunk.7z &
+      7z x C:\ruby_trunk.7z -oC:\ruby_trunk & C:\ruby_trunk\trunk_install.cmd)
 environment:
   matrix:
     - RUBY_VERSION: 23-x64
+    - RUBY_VERSION: 24-x64
+    - RUBY_VERSION: _trunk
```

Comment:
Any reason to call this `_trunk` and not `trunk`?

Reply:
I guess I preferred a folder name of ruby_trunk instead of rubytrunk.  That's the only reason, as something like ruby23 seems appropriate, but...

Reply:
Ah, makes sense, that's fine :)